### ISSUE TITLE
`indentation` writes an indent into a single-line styled template standing on an indented host line

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#453]
`indentation` writes an indent into a single-line styled template whose host line is indented: the expected indentation of the template's first node adds the host line's level even where the node shares that very line, so the fix puts the host's tab inside the template.

### What happened

For a node of a styled template, `indentation` adds the level of the host line the template's expression opens on. For a template broken over lines that is right — its content stands on lines of its own. For a single-line template the content stands on the host's own line, there is no indentation of the stylesheet to speak of, and the expected value should be empty; the first-child branch still compares against the host level and the fix writes that many tabs into the template.

### What minimal example or steps are needed to reproduce the bug?

```js
function f () {
	const a = styled.div`color: red;`;
}
```

Linting reports `Expected indentation of 0 tabs` on `color: red;`, and `--fix` rewrites the template to `` styled.div`	color: red;` `` — a tab inside the template. On the namespace stack the rule is `@stylistic/styled/indentation`; on `main` today the same branch fires under `@stylistic/indentation`.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"customSyntax": "postcss-styled-syntax",
	"rules": { "@stylistic/indentation": ["tab"] }
}
```
